src/FunderMaps.WebApi/Controllers/Report/RecoveryController.cs

```diff
@@ -96,6 +95,7 @@ public async Task<IActionResult> UploadDocumentAsync([Required] IFormFile input)
             }
 
             // FUTURE
+            await Task.CompletedTask;
```

Comment:
Why is this not yet implemented? All our infrastructure exists for uploading documents. Otherwise, does there exist an issue for this matter?

Reply:
#341 created.